consensus/hotstuff/pacemaker/pacemaker_test.go

```diff
@@ -30,456 +32,188 @@ func expectedTimerInfo(view uint64, mode model.TimeoutMode) interface{} {
 		})
 }
 
-func expectedTimeoutInfo(view uint64, mode model.TimeoutMode) interface{} {
-	return mock.MatchedBy(
-		func(timerInfo *model.TimerInfo) bool {
-			return timerInfo.View == view && timerInfo.Mode == mode
-		})
+func TestActivePaceMaker(t *testing.T) {
```

Comment:
Suggesting some additional test cases:
* `TestProcessTC_IgnoreOldTC` - same as `TestProcessQC_IgnoreOldQC`, but for TCs
* `TestProcessTC_IgnoreNilTC` - test that a nil TC is an allowable input
* Process a TC, then a QC -> make sure `LastViewTC` is `nil`